x/stakeibc/keeper/update_redemption_rates_test.go

```diff
@@ -88,18 +78,17 @@ func (s *KeeperTestSuite) TestUpdateRedemptionRatesSuccessful() {
 	tc := s.SetupUpdateRedemptionRates(stakedBal, undelegatedBal, justDepositedBal, stSupply, redemptionRateT0)
 
 	// sanity check on inputs (check redemptionRate at genesis is 1)
-	s.Require().Equal(tc.redemptionRateT0, sdk.NewDec(1))
+	s.Require().Equal(redemptionRateT0, sdk.NewDec(1), "to rr")
```

Comment:
is this supposed to be "to" or "t0" (as in time zero)?
Maybe just rename it to `initialRedemptionRate`?

Reply:
done